### PR TITLE
#16 BackEnd Implementation (SignIn)

### PR DESCRIPTION
**Pull request by Electryk**
Added SignIn.java
Updated web.xml

### CODE REVIEW

**Comment by Gascon95** [comment]
@nebur395 means that, in the SignIn servlet, you are returning to the client all the information about the user (nick, pass, email and date), but the API documentation specifies that only the nick and the pass must be returned. Remember to check it out on GoogleDrive.

**Comment by Gascon95** [comment]
@Electryk you still have to send an error message on the SignIn servlet when the login process fails.

File: src/main/java/org/trainingTracker/servlets/SignIn.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                UserVO vo = UsersDAO.findUser(name);
+                // If user don't exists
+                if (vo==null){
+                    response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+                    response.getWriter().println("El usuario " + name + " no existe");
+                }
+                // If pass don't match
+                else if(!(vo.getPass()).equals(pass)){
+                    response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+                    response.getWriter().println("Password incorrecto");
```
Comment (nebur395) [inline src/main/java/org/trainingTracker/servlets/SignIn.java]: Try to keep a standard in your messages and do not mix languages:
response.getWriter().println("Contraseña incorrecta");

**Comment by nebur395** [comment]
You are still returning wrong information to the client side. Please, take a look again to our API. At this time, you are returning the next JSON object:
```
{
    "Nickname": "XXX",
    "Mail": "XXX"
}
```
but we need something like this:
```
{
    "user": "XXX",
    "email": "XXX"
}
```

File: src/main/java/org/trainingTracker/servlets/SignIn.java (hunk 1 above)
Comment (Gascon95) [inline src/main/java/org/trainingTracker/servlets/SignIn.java]: @Electryk, you still have to make this changes.

**Pull request merged**